src/providers/fault-mgmt-providers/realtime-fault-provider.js

```diff
@@ -34,13 +33,13 @@ export default class RealtimeFaultProvider {
     }
 
     subscribe(domainObject, callback) {
-        const globalUnsubscribe = this.#openmct.telemetry.subscribe(
+        const globalUnsubscribe = this.realtimeProvider.subscribe(
```

Comment:
why can't we use the openmct API methods here?

Reply:
The Open MCT API looks for the provider and calls the provider.subscribe method, The provider in this case is the realtime fault management provider. So pointing it back to the Open MCT API would not make any sense.

Reply:
That's how consumers are supposed to use the API though, right? We should be dogfooding where possible. @akhenry thoughts ?